engine/src/main/java/org/camunda/bpm/engine/impl/batch/deletion/DeleteProcessInstanceBatchConfigurationJsonConverter.java

```diff
@@ -49,9 +54,13 @@ public JsonObject toJsonObject(DeleteProcessInstanceBatchConfiguration configura
 
   public DeleteProcessInstanceBatchConfiguration toObject(JsonObject json) {
     DeleteProcessInstanceBatchConfiguration configuration =
-      new DeleteProcessInstanceBatchConfiguration(readProcessInstanceIds(json), null, JsonUtil.getBoolean(json, SKIP_CUSTOM_LISTENERS), JsonUtil.getBoolean(json, SKIP_SUBPROCESSES), JsonUtil.getBoolean(json, FAIL_IF_NOT_EXISTS));
+      new DeleteProcessInstanceBatchConfiguration(readProcessInstanceIds(json), readIdMappings(json), null,
+          JsonUtil.getBoolean(json, SKIP_CUSTOM_LISTENERS), JsonUtil.getBoolean(json, SKIP_SUBPROCESSES),
+          JsonUtil.getBoolean(json, FAIL_IF_NOT_EXISTS));
 
     String deleteReason = JsonUtil.getString(json, DELETE_REASON);
+    deleteReason = JsonUtil.getString(json, DELETE_REASON);
```

Comment:
?

Reply:
Don't know what happened there, thanks for noticing, I'll remove that. 👍